packages/snap-tracker/src/Tracker.ts

```diff
@@ -211,6 +212,45 @@ export class Tracker {
 			return beaconEvent;
 		},
 
+		error: (data: TrackErrorEvent, siteId?: string): BeaconEvent | undefined => {
+			if (!data?.stack && !data?.message) {
+				// no console log
+				return;
+			}
+			let context = this.context;
+			if (siteId) {
+				context = deepmerge(context, {
+					context: {
+						website: {
+							trackingCode: siteId,
+						},
+					},
+				});
+			}
+			const payload = {
+				type: BeaconType.ERROR,
+				category: BeaconCategory.METRICS,
+				context,
+				event: {
+					type: data?.type || 'error',
+					userAgent: data?.userAgent || navigator.userAgent,
+					href: data?.href || window.location.href,
+					siteId: data?.siteId || context.website.trackingCode,
+					framework: data?.framework,
+					version: data?.version || version,
+					filename: data?.filename,
+					stack: data?.stack,
+					message: data?.message,
+					colno: data?.colno,
+					lineno: data?.lineno,
+					timeStamp: data?.timeStamp,
```

Comment:
I wonder if the timestamp / date here is redundant - we don't send those with other beacons.

Reply:
I removed date. Keeping timeStamp as in this context it contains the elapsed time since page load and when the error occurred